py3status/modules/deadbeef.py

```diff
@@ -67,56 +85,40 @@ def _is_running(self):
 
     def deadbeef(self):
         color = self.color_stopped
-        artist = isplaying = length = playback_time = title = \
-            tracknumber = year = ""
+        status = None
 
         if not self._is_running():
             return {
                 'cached_until': self.py3.time_in(self.cache_timeout),
-                'color': color,
-                'full_text': ''
+                'full_text': self.py3.safe_format(self.format),
+                'color': color
```

Comment:
This code feels like it is just repetition, we have the same lines again in a moment, can't we share them?